src/node_contextify.cc

```diff
@@ -328,23 +342,27 @@ BaseObjectPtr<ContextifyContext> ContextifyContext::New(
     result->MakeWeak();
   }
 
-  if (sandbox_obj
+  Local<Object> referrer =
+      sandbox_obj.IsEmpty() ? new_context_global : sandbox_obj;
+  if (!referrer.IsEmpty() &&
+      referrer
```

Comment:
If I understand correctly, this `referrer` just means it is a GC referrer to the contextify wrapper object instead of module referrer?

I think the name `referrer` is heavily used in module resolution and just a line below we are initializing module host defined option, here it is a bit confusing.